packages/@react-spectrum/provider/test/Provider.test.js

```diff
@@ -110,6 +114,20 @@ describe('Provider', () => {
     expect(provider2.classList.contains('spectrum--light')).toBeTruthy();
   });
 
+  it('Nested providers pass props to children', () => {
+    let {getByRole} = render(
+      <Provider theme={theme} isDisabled>
+        <Provider isQuiet>
+          <Button onPress={onPressSpy}>Hello!</Button>
```

Comment:
if the onPressSpy is only used in this test, then just declare it here in this 'it' instead of at the top, then it'll get cleaned up automatically